airbyte-integrations/connectors/tasks.py

```diff
@@ -111,7 +126,7 @@ def _run_task(ctx: Context, connector_string: str, task_name: str, multi_envs: b
 
     commands = []
 
-    commands.extend([cmd.format(source_path=source_path, venv=venv_name, **kwargs) for cmd in TASK_COMMANDS[task_name]])
+    commands.extend([cmd.format(source_path=source_path, venv=venv_name, **kwargs) for cmd in task_commands[task_name]])
```

Comment:
shouldn't it be upper case?

Reply:
nah, I've changed that to be able to override task commands per `_run_task ` call